scripts/run-webservice-script.sh

```diff
@@ -10,5 +10,5 @@ psql -h localhost -c "create user dockstore with password 'dockstore' createdb;"
 psql -h localhost -c "ALTER USER dockstore WITH superuser;" -U postgres
 psql -h localhost -c 'create database webservice_test with owner = dockstore;' -U postgres
 psql -h localhost -f travisci/db_dump.sql webservice_test -U postgres
-java -jar dockstore-webservice.jar db migrate -i 1.5.0,1.6.0,1.7.0 travisci/web.yml
+java -jar dockstore-webservice.jar db migrate -i 1.5.0,1.6.0,1.7.0,add_service_1.7.0 travisci/web.yml
```

Comment:
Just double checking, but won't this add the test service to all tests whether they're testing services or not?

Reply:
ya, but it shouldn't really be a problem.  Our initial DB also adds tools/workflows for all tests

Reply:
Ok, just making sure it was intentional